src/graphql/donor/index.tsx

```diff
@@ -0,0 +1,26 @@
+import { gql } from "@apollo/client";
+
+export const GET_ORG_DONOR = gql`
+	query getOrgDonorByOrg($sort: String, $limit: Int, $start: Int, $filter: JSON) {
```

Comment:
indentation doesn't look right

Reply:
I have pressed ctrl+shift+i it is still the same